tests/helm_transform_tests.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+# Require binaries in $PATH to run helm transform script
+which kpt
+which helm
+
+# Clone helm charts repository for example charts, fails if charts are already cloned
+git clone git@github.com:helm/charts.git
+
+# Helm transform script tests
+# Works as source function
+sh/src/helm_transform.sh charts/stable/redis | kpt config tree
```

Comment:
Let's remove the non-dockerized ones (since catalog is all docker images). It's distracting from the section below.